// Copyright 2015 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//

syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package blimp;

message RenderWidgetMessage {
  enum Type {
    // TODO(dtrainor): Leaving this a bit open in case we need to send other
    // types of RenderWidget messages (like shutdown).

    // Engine => Client types.
    INITIALIZE = 1;
  }

  optional Type type = 1;
  optional uint32 render_widget_id = 2;
}